src/locales/en_US.js

```diff
@@ -90,6 +90,7 @@ const strings = {
     'Stellar (XLM) wallets require a 1 XLM minimum balance. You must deposit at least 1 XLM to this address before this wallet will show a balance or transactions. 1 XLM will be unspendable for the lifetime of this wallet address.',
   fragment_send_address_dialog_title: 'Send to Public Address',
   fragment_send_address_dialog_title_short: 'Send to Address',
+  fragment_send_transfer: 'Transfer',
```

Comment:
Please use "fragment_transaction_transfer" instead. This prevents additional translation requirements